src/views/auth/Signout.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <sbc-signout :redirect-url="redirectUrl"></sbc-signout>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Vue } from 'vue-property-decorator'
+
+// Components
+import SbcSignout from 'sbc-common-components/src/components/SbcSignout.vue'
+
+@Component({
+  components: {
+    SbcSignout
+  }
+})
+export default class SignoutView extends Vue {
+  private redirectUrl = `${window.location.origin}/${process.env.VUE_APP_PATH}`
```

Comment:
I suggest you redirect to `AUTH_URL` instead. (Specifically a configured value instead of an assumed calculated URL.)

Reply:
On signout, the user should be sent back to the Business Registries home page, which will be bcregistry.gov.bc.ca/business but is currently https://dev.bcregistry.ca/cooperatives/auth/.

Reply:
As per our discussion, we'll look at correcting this in a (near) future ticket.

Reply:
No, don't send user back to our app (which will then redirect to the signin page).

Instead, send them "up" to the Business Registries home page (per comment above).